scale/docs/rest/v6/job.rst

```diff
@@ -630,6 +631,8 @@ Response: 200 OK
 | error              | JSON Object       | The error that is associated with the job.                                     |
 |                    |                   | (See :ref:`Error Details <rest_v6_error_details>`)                             |
 +--------------------+-------------------+--------------------------------------------------------------------------------+
+| max_tries          | Integer           | Number of times this a job of this type will be automatically retried.         |
```

Comment:
minor typo. I think you meant "Number of times this job will be automatically retried."

Reply:
Good eye. Fixed with better wording.